bob/global/env.go

```diff
@@ -1,4 +0,0 @@
-package global
-
-// EnvWhitelist keep the whitelisted env variables when hermetic mode is on
-var EnvWhitelist = []string{"HOME", "XDG_CACHE_HOME"}
```

Comment:
It looks to me that XDG_CACHE_HOME is not kept

![image](https://user-images.githubusercontent.com/3776893/200116874-e599a5eb-6664-4d34-8566-3f7fd06f392b.png)

Reply:
I removed our implementation of whitelisting env variables and relied on `nix-shell --pure` whitelisting strategy:

https://github.com/NixOS/nix/blob/c0559a1d6092b94c7ce46a065d72fc74e4db51dd/src/nix-build/nix-build.cc#L100

If there is a reason to keep XDG_CACHE_HOME I can revert and add it back

Reply:
Marking as resolved:

"Yes i think there was a reason for it
I remember go uses this to reuse the cache on the users home 
Please keep it"